test_scripts/API/VehicleData/StabilityControlsStatus/008_GetVehicleData_StabilityControlsStatusNotAllowedByLolicy.lua

```diff
@@ -0,0 +1,55 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@ydementieiev please check the script name test_scripts/API/VehicleData/StabilityControlsStatus/008_GetVehicleData_StabilityControlsStatusNotAllowed**ByLolicy.lua**

Reply:
@LitvinenkoIra thank you, [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/14/commits/caa2613faa5f68d44873fbeea3d23d1a84022d10)

Reply:
@LitvinenkoIra I like `Lolicy` :smile: